pkgs/misc/ptouch-print/default.nix

```diff
@@ -1,33 +1,49 @@
-{ lib, stdenv
+{ lib
```

Comment:
for consistency these should probably be alphabetical. They weren't before, but seeing how much you change it'd make sense to fix this.